packages/runtimes/js/src/JSRuntime.js

```diff
@@ -65,8 +70,52 @@ let bundleDependencies = new WeakMap<
   |},
 >();
 
+type JSRuntimeConfig = {|
+  splitManifestThreshold: number,
+|};
+
+let defaultConfig: JSRuntimeConfig = {
+  splitManifestThreshold: 100000,
```

Comment:
How did we determine 100kb as the threshold?

Reply:
As discussed, not a lot of thought went into this and it's more of a "finger in the air" guess. We'll merge and potentially refine it later after some tests.